integration-testing/src/main.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+use ipc_sdk::subnet_id::SubnetID;
+use std::str::FromStr;
+use std::sync::atomic::AtomicU16;
+use std::sync::Arc;
+use std::thread::sleep;
+use std::time::Duration;
+
+mod infra;
+
+#[tokio::main]
+async fn main() {
```

Comment:
Would it make sense to make this into a CLI so we can pass the tests that we want to run, flags to config the infra deployment, etc.?